### ISSUE TITLE
Add image into the first frame

### ISSUE THREAD

**Issue by cdfan0627** [THUDM/CogVideo#650]
請問為什麼在 i2v_dataset.py 中，需要將image condition 加入到video frames中，如下code，image condition不是本來就是複製first frames得到的，然後video frames不是本身就有包含first frames了嗎，這樣子video frames 會等於有兩個first frames吧
`frames = torch.cat([image.unsqueeze(0), frames], dim=0)`
